wallets/README.md

```diff
@@ -0,0 +1,86 @@
+# Wallets
+
+## Wallet object
+
+```
+{
+  userId: <userId>,
+  isActive: bool,
+  currencies: {
+    neelam: number,
+    dinero: number
+  }
+}
+```
+
+## **Requests**
+
+|                     Route                     |                  Description                   |
+| :-------------------------------------------: | :--------------------------------------------: |
+|          [GET /wallet](#get-wallet)           |        Returns  the user wallet details        |
+| [GET /wallet/:username](#get-walletusername) | Returns the user wallet details for a username |
+
+
+## **GET /wallet**
+
+Returns  the user wallet details
```

Comment:
Can we be more explicit here in mentioning that this route will return the logged in user their own wallet?